discord/webhook.py

```diff
@@ -400,6 +401,16 @@ async def foo():
         webhook = Webhook.partial(123456, 'abcdefg', adapter=RequestsWebhookAdapter())
         webhook.send('Hello World', username='Foo')
 
+    .. container:: operations
```

Comment:
Should this container have a versionadded text?

Reply:
Sorry, I'm not familiar with what that is. Can I see an example or an explanation? ^_^"

Reply:
Something like 
```
.. versionchanged:: 1.4.0
     Webhooks now extend hashable.
```

Reply:
thx